README.md

```diff
@@ -4,227 +4,211 @@
+Make sure to read the corresponding Swagger-generated documentation of this
+operations, located in `docs/active_docs/Service Management API.json`
 
-> $ `bundle install`
+It attempts to respond with lowest possible latency, performing minimal work
+and offloading other work to Apisonator workers by enqueuing tasks into job queues.
 
-The container image has additional tools to handle these dependencies for
-multiple Ruby versions. Check out the `scripts` directory if you are curious.
+This component needs access to a Redis database to perform the following actions:
+ * Enqueue reports, which will be processed by the Apisonator worker component
+ * Perform authorization of the requests
+These two actions can be configured to be performed in different Redis
+databases if desired (see the Prerequisites and Configuration sections).
 
-#### Workflow
+Finally, another API named 'Internal API' is provided to configure services
+in Apisonator. This API is intended only for administrative purposes and not
+for general consumption. Therefore, usage of this API should be protected or
+not exposed to untrusted parties. You can also generate its documentation with
+Rake tasks. The [Pisoni](https://github.com/3scale/pisoni) API client can be
+used to interact with the Internal API.
 
-Your project directory within the container is sync'ed with your local clone of
-the project so that changes in one reflect instantly in the other.
+To quickly test Apisonator, random services can be created and configured on it
+via the use of the 'Buddhi' tool located in our performance tests
+toolkit: [3scale perftest-toolkit](https://github.com/3scale/perftest-toolkit/).
 
-We recommend editing code and committing locally, and executing tests within the
-container, since the container won't have your own tools and configurations.
 
-### On local machine (unsupported)
+This component may also be referred to as '3scale_backend'.
 
-This is **unsupported** and **strongly discouraged** because it is the source of a lot
-of headaches and potential problems and introduces the need for us to document
-dependencies and config files here that are very likely to be outdated and
-incomplete.
+### Apisonator worker
 
-Bear in mind that we can only provide best effort support for this.
+It is responsible for performing background tasks off-loaded from
+Apisonator listeners (enqueued jobs).
 
-1. Install Redis v2.8.19 (or whichever version is recommended for use).
-2. Run `bundle install`.
-3. Configure Redis in `~/.3scale_backend.conf` as:
+The worker component takes care of running these enqueued jobs, mainly related
+to reporting of previous traffic.
 
-```
-ThreeScale::Backend.configure do |config|
-  config.redis.proxy = 'localhost:6379'
-end
-```
+This component needs access to a Redis database to perform the following actions:
+ * Dequeue the report jobs that have been submitted to the Redis database by the
```

Comment:
Maybe explanation about what to do with dequeued jobs?

Reply:
I'm not sure what you want to mean here. This is automatically performed by the Apisonator Workers. There is nothing "to do" with the dequeued jobs by the user.

Do you mean, "explain what is done with the dequeued jobs"?